### PR TITLE
Russian and Hindi config updates

### LINKED ISSUE #5969: Hindi, Mundo, Russian review
**Is your feature request related to a problem? Please describe.**
We want to release Hindi, Mundo and Russian front pages. We should check all translations, configuration, and links. Similar to the work in this issue: https://github.com/bbc/simorgh/issues/5961

**Describe the solution you'd like**
- We should check requirements with Jon.
- We should check the service config as well as markup of the page to ensure we have translations for all offscreen text as well as on screen text.
- We should check Chartbeat config, ATI config, consent banners, metadata, navigation links, footer links, trust project links. Note that this should be done for these pages:

https://www.test.bbc.com/hindi
https://www.test.bbc.com/mundo
https://www.test.bbc.com/russian

- If there are any bugs, they should be fixed.
- Then we should review with UX designers (Laura and Ed).

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Unit tests should be sufficient, plus manual checks.

- [ ] This feature is expected to need manual testing.

**Additional context**

### LINKED ISSUE #6049: Update trust links for Russian, Mundo and Hindi
**Is your feature request related to a problem? Please describe.**
Update the trust links for the following sites.
Only merge when all the pages work. This ticket will be updated 

**Describe the solution you'd like**

Site | Link | Link works
-- | -- | --
Hindi | https://www.bbc.com/hindi/institutional-50223932 | Y
Mundo | https://www.bbc.com/mundo/institucional-51359666 | Y
Russian | https://www.bbc.com/russian/institutional-50098149 | N

Only merge when all the pages are working.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by OlgaLyubin**
Resolves #6049 and #5969

**Overall change:** 
Updated Russian and Hindi configs for live release of the services.

**Do not merge** 
Russian trust project link has not been published yet, so need to wait for it to be working before merging this PR.

**Code changes:**
- Updated Russian and Hindi configs with translations.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/lib/config/services/russian.js [hunk 1]
```diff
@@ -44,8 +44,9 @@ export const service = {
     themeColor: `${C_POSTBOX}`,
     twitterCreator: '@bbcrussian',
     twitterSite: '@bbcrussian',
-    noBylinesPolicy: 'https://www.bbc.com/news/help-41670342#authorexpertise',
-    publishingPrinciples: 'https://www.bbc.com/news/help-41670342',
+    noBylinesPolicy:
+      'https://www.bbc.com/russian/institutional-50098149#authorexpertise',
+    publishingPrinciples: 'https://www.bbc.com/russian/institutional-50098149',
```
Comment (EinsteinNjoroge) [inline src/app/lib/config/services/russian.js]: @OlgaLyubin all the new links in the Russian service redirect to 404 pages, is that expected or should the relevant articles be published first?

Comment (OlgaLyubin) [reply src/app/lib/config/services/russian.js]: They should be published first. I will add a "do not merge" to this PR's name and will not merge it before all the links are working:)

**Review by Bopchy — APPROVED** [review]

File: src/app/lib/config/services/russian.js (hunk 1 above)
Comment (EinsteinNjoroge) [reply src/app/lib/config/services/russian.js]: @OlgaLyubin Might be good to add a note to the description as well just for visibility.

**Review by EinsteinNjoroge — APPROVED** [review]

**Comment by OlgaLyubin** [comment]
I have deleted Russian trust project link from this PR because it is not going to be published before the live release tomorrow. However, editorial is happy to release Russian with the English trust link.

New PR for the Russian trust project link that will be merged later when the link gets published: https://github.com/bbc/simorgh/pull/6069

**Pull request merged**